### PR TITLE
Config with pure python

### PR DESCRIPTION
**Pull request by icemac**
* Lint the code.
* Add support for Python 3.9.

### CODE REVIEW

File: src/zodbupdate/tests.py [hunk 1]
```diff
@@ -216,10 +218,10 @@ def tearDown(self):
 
     def _tearDownStorage(self):
         """
-        Called during *tearDown* to clean up storage-specific artifacts.
+        Called during *tearDown* to clean up storage-specific artefacts.
```
Comment (dataflake) [inline src/zodbupdate/tests.py]: Fun fact: In US English "artifact" is correct. Artefact is correct for British English.

**Review by dataflake — APPROVED** [review]

**Comment by icemac** [comment]
I added back `py27-oldzodb` but it breaks with `AttributeError: 'TransactionMetaData' object has no attribute 'extension_bytes'`. This this tox environment still needed, so we should fix it?

**Comment by dataflake** [comment]
That `AttributeError` usually shows up when combining storages that have been updated for ZODB 5.6 with those that haven't.  What is that test environment really testing, and is that a valid use case here?

**Comment by icemac** [comment]
At least we have separate tests paths for ZODB < 5.4 and newer versions. So the code still seems to support ZODB < 5.4.

**Comment by dataflake** [comment]
I guess my question is this: Why do we need to test with these old versions of ZODB? Why would you run a conversion on those versions, as opposed to doing that conversion in a dedicated sandbox under Zope 4/Python 2?

**Comment by icemac** [comment]
> I guess my question is this: Why do we need to test with these old versions of ZODB? Why would you run a conversion on those versions, as opposed to doing that conversion in a dedicated sandbox under Zope 4/Python 2?

I am not sure. As I found a working combination of versions, we can keep it for now until we drop Python 2 support.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**